src/About/SupportCard.tsx

```diff
@@ -60,7 +61,14 @@ export default () => {
                         label="Create system report"
                     />
                 </Section>
-                <Section title="Verbose Logging">
+                <Section title="Other feedback">
+                    <p>
+                        Provide feedback about how to improve this and other nRF
+                        Connect for Desktop applications.
```

Comment:
Can we avoid breaking the `nRF Connect for Desktop` name over two lines? Would something like this work?
```suggestion
                        Provide feedback about how to improve this and other</br>nRF</nbsp>
                        Connect for Desktop applications.
```

Reply:
The way the text breaks here and the way it breaks in the UI are not the same.
In the UI it already respects your wish
![Screenshot from 2024-03-25 12-33-46](https://github.com/NordicSemiconductor/pc-nrfconnect-shared/assets/30871542/4eb7fd0c-722c-421d-b2d0-d64f8f0ca0a2)